erdgo/proxy.go

```diff
@@ -0,0 +1,209 @@
+package erdgo
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+const (
+	proxyURL    = "http://174.138.103.62:8079" // "https://api-testnet.elrond.com"
+	metachainID = 4294967295
+	// endpoints
+	networkConfigEndpoint        = "network/config"
+	accountEndpoint              = "address/%s"
+	sendTransactionEndpoint      = "transaction/send"
+	getTransactionStatusEndpoint = "transaction/%s/status"
+	getTransactionInfoEndpoint   = "transaction/%s"
+	getHyperblockByNonceEndpoint = "hyperblock/by-nonce/%v"
+	getHyperblockByHashEndpoint  = "hyperblock/by-hash/%s"
+	getNetworkStatusEndpoint     = "network/status/%v"
+)
+
+// GetNetworkConfig retrieves the network configuration from the proxy
+func GetNetworkConfig() (*NetworkConfig, error) {
```

Comment:
How about attaching these functions to a new struct called `ElrondProxy`?

Reply:
Done